ext_tables.sql

```diff
@@ -125,7 +125,7 @@ CREATE TABLE tx_t3kit_accordion_item (
 #
 CREATE TABLE sys_file_reference (
 	description_position varchar(60) DEFAULT '' NOT NULL,
-	picture_width varchar(60) DEFAULT '' NOT NULL,
+	picture_width smallint(100) DEFAULT '' NOT NULL,
```

Comment:
I think we need to add `DEFAULT 0`
`picture_width smallint(100) DEFAULT 0 NOT NULL,`

Reply:
Fair point, PR is updated.